java/ql/lib/ext/experimental/io.undertow.server.handlers.resource.model.yml

```diff
@@ -1,8 +0,0 @@
-extensions:
-  - addsTo:
-      pack: codeql/java-all
-      extensible: experimentalSummaryModel
-    data:
-      - ["io.undertow.server.handlers.resource", "Resource", True, "getFile", "", "", "Argument[this]", "ReturnValue", "taint", "manual", "unsafe-url-forward"]
```

Comment:
Don't we want these models anymore?

Reply:
I've added all resource-related models to a separate PR (https://github.com/github/codeql/pull/15921). Is it okay if I add them on that PR instead?

See my note from the PR description above:

> Note: this promotion does not include the resource-related additions in https://github.com/github/codeql/pull/8706 and https://github.com/github/codeql/pull/9199. Since we already model some of these resource methods ([example](https://github.com/github/codeql/blob/9c51514bd9fdd877990cb7a62a3688127002c392/java/ql/lib/ext/java.lang.model.yml#L6-L7)) as path-injection sinks, it probably makes more sense to add them to the path-injection query instead (see https://github.com/github/codeql/pull/15921). Let me know if you think any of these models should be added back to this query instead.

(sorry, re-reading my note, I did not make it clear that I had put resource-related taint steps as well as sinks in https://github.com/github/codeql/pull/15921).

Reply:
Ah, my bad, I hadn't read the description of this PR in a while. It's perfectly fine for this models to be in another PR 👍.